benchmark/Resp.benchmark/ReqGenUtils.cs

```diff
@@ -95,10 +85,19 @@ private bool WriteOp(ref byte* curr, byte* vend, OpType opType)
                         return false;
                     break;
                 case OpType.BITOP_AND:
+                    if (!WriteStringBytes(ref curr, vend, Encoding.ASCII.GetBytes("AND")))
```

Comment:
This would be more efficient as it avoids calling GetBytes every time:

```
while (!RespWriteUtils.WriteDirect(CmdStrings.AND, ref dcurr, dend))
    SendAndReset();
```
and
```
public static ReadOnlySpan<byte> AND => "AND"u8;
```

But also note that is just pre-generation of data in client benchmark, so it is not perf critical.